tests/dummy/app/templates/default-value.hbs

```diff
@@ -0,0 +1,65 @@
+<p>This example shows how you can set default values for your selections by setting the selected=true property on an option and how these defaults are impacted by changes to the dynamic content in your list of options.</p>
+<hr>
+<h2>Select a Car Make</h2>
+<p>Default set to option two (Honda)</p>
+<p>
+  {{#x-select value=make action="setMake" class="spec-car-make"}}
```

Comment:
alternatively:

```hbs
{{#x-select value=(mut make) action=(action "setMake")}}
```

If you want to still call the `setMake` action in the controller.